app/controllers/reports_controller.rb

```diff
@@ -37,6 +37,13 @@ def value_by_county
     @report = Reports::ValueByCounty.new(params, session)
   end
 
+  def price_point_variance
+    purchases = Purchase.includes(:vendor, :purchase_details)
+    start_date = params[:report_start_date].present? ? Date.strptime(params[:report_start_date], "%m/%d/%Y") : Date.new(2001, 1, 1)
+    end_date = params[:report_end_date].present? ? Date.strptime(params[:report_end_date], "%m/%d/%Y") : Date.today
+    @purchases = purchases.where(purchase_date: start_date..end_date)
```

Comment:
It looks like you aren't using the report object you created in `app/models/reports/price_point_variance.rb`